bin/next-start

```diff
@@ -38,7 +39,14 @@ if (argv.help) {
 const dir = resolve(argv._[0] || '.')
 
 const srv = new Server({ dir })
-srv.start(argv.port)
+
+exists(resolve(dir, '.next')).then(exists => {
```

Comment:
fs.exists is a deprecated API. Use `fs.exitsSync`.
Import if from `fs` directly.